proxies/ruby/app.rb

```diff
@@ -19,49 +23,89 @@
 
 data_store = {
   clients: {},
-  commandResults: {}
+  command_results: {}
 }
 
 post '/client' do
   request.body.rewind
-  data = JSON.parse request.body.read
+  body = JSON.parse request.body.read
 
   client_id, sdk_key, wait_for_initialization, options =
-    data.values_at('clientId', 'sdkKey', 'waitForInitialization', 'options')
+    body.values_at('clientId', 'sdkKey', 'waitForInitialization', 'options')
 
   if client_id.nil?
     status 400
     return { message: 'Invalid request: missing clientId' }.to_json
   end
 
   begin
-    options = DevCycle::DVCOptions.new(
-      config_cdn_uri: options.fetch('configCDNURI', 'https://config-cdn.devcycle.com'),
-      events_api_uri: options.fetch('eventsAPIURI', 'https://events.devcycle.com'),
+    dvc_options = DevCycle::DVCOptions.new(
+      config_cdn_uri: options.fetch('configCDNURI', ''),
```

Comment:
why are we defaulting to empty string now?

Reply:
there's no point in defaulting to our actual config CDN and events api. We're not using real sdk keys for the test harness so we'd be making requests just to have them fail

Reply:
we also really do not want this thing making real requests by default